io.github.cmus.cmus.metainfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
```

Comment:
```suggestion
<component type="console-application">
```

Reply:
I thought we were fine with calling it `desktop-application` if it's more TUI than CLI? We have done it already for Poezio and something Bitcoin related I think.

Reply:
Although, interestingly, it seems to retain icons and screenshots in app-info, I guess thanks to the desktop file. It won't be displayed on the website though.

Reply:
This is definitely more TUI than CLI. Neovim for one uses `desktop-application` as its type as well (although that's from its upstream instead of being Flathub specific).

I'll switch it back to desktop application then, also the launch button in GNOME Software doesn't work (I guess since a .desktop file wouldn't apply to console applications normally).